config/formats.js

```diff
@@ -570,15 +562,15 @@ let Formats = [
 
 		mod: 'gen7',
 		gameType: 'doubles',
-		searchShow: false,
 		forcedLevel: 666,
 		teamLength: {
 			validate: [2, 2],
 			battle: 2,
 		},
-		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		searchShow: false,
+		ruleset: ['Pokemon', 'HP Percentage Mod', 'Cancel Mod'],
```

Comment:
The banlist was based on http://spo.ink/shitmons which doesn't ban CAP mons, only Kitsunoh specifically. `Pokemon` means that none of the legal CAPs can be used either.

Reply:
Interesting. I guess change it back if we’re following that completely.